apps/src/aiComponentLibrary/copyButton/copy-button.module.scss

```diff
@@ -1,10 +1,10 @@
-@import 'color';
+@import '@cdo/apps/componentLibrary/common/styles/colors.css';
```

Comment:
@levadadenys should I be getting colors from the `component-library-styles` package? I also have a mixin update/import below, but realizing mixins now live in that package as well, so I'll probably move this mixin to live in `code-dot-org` since it's not part of the design system.

Reply:
You should use '@code-dot-org/component-library-styles/colors.scss'.

`@code-dot-org/component-library-styles` is part of design system, we just had to move styles in scope of optimisation so to say

Yeah, I think for now `icon-button` mixin live in `apps` directory. It's possible that we reuse it later when working on IconToggleButton, but I agree for now it makes sense to keep it in `/apps/src/...` for now.

Reply:
@levadadenys I added a dependency on `@code-dot-org/component-library-styles` in `apps` to support this (looks like we hadn't done this yet?), LMK if it looks ok!

Reply:
Discussed with Denys, going to wait for this PR to be merged, which adds the dependency (and remove adding it here): https://github.com/code-dot-org/code-dot-org/pull/63865